recipes/grpc/config.yml

```diff
@@ -0,0 +1,11 @@
+versions:
+  "1.25.0":
+    folder: all
+  "1.26.0":
+    folder: all
+  "1.27.0":
+    folder: all
+  "1.27.2":
+    folder: all
+  "1.27.3":
+    folder: all
```

Comment:
I would recommend adding only one version to begin (the latest for example), to quickly check errors with CI.